react/components/molecules/card/card.js

```diff
@@ -0,0 +1,152 @@
+import React, { PureComponent } from "react";
+import { StyleSheet, Text, View, ViewPropTypes } from "react-native";
+import PropTypes from "prop-types";
+
+import { Avatar, Icon, Touchable } from "../../atoms";
+
+import { baseStyles } from "../../../util";

```

Comment:
Move this above atoms import. Normally we order by how "external" and "removed" it is to the current directory.